x-pack/plugins/maps/public/shared/layers/sources/es_geo_grid_source/es_geo_grid_source.js

```diff
@@ -66,6 +66,11 @@ export class ESGeoGridSource extends AbstractESSource {
 
   static renderEditor({ onPreviewSource, inspectorAdapters }) {
     const onSelect = (sourceConfig) => {
+      if (!sourceConfig) {
+        onPreviewSource();
```

Comment:
I'd pass in `null`. This signature overloading looks funny, especially compared to where we use this with a valid source argument.